cmake/ci.cmake

```diff
@@ -551,6 +551,20 @@ add_custom_target(ci_test_diagnostics
     COMMENT "Compile and test with improved diagnostics enabled"
 )
 
+###############################################################################
+# Enable diagnostic positions support.
+###############################################################################
+
+add_custom_target(ci_test_diagnostic_positions
+    COMMAND ${CMAKE_COMMAND}
+    -DCMAKE_BUILD_TYPE=Debug -GNinja
+    -DJSON_BuildTests=ON -DJSON_Diagnostics=ON -DJSON_Diagnostic_Positions=ON
+    -S${PROJECT_SOURCE_DIR} -B${PROJECT_BINARY_DIR}/build_diagnostics_positions
+    COMMAND ${CMAKE_COMMAND} --build ${PROJECT_BINARY_DIR}/build_diagnostics_positions
+    COMMAND cd ${PROJECT_BINARY_DIR}/build_diagnostics_positions && ${CMAKE_CTEST_COMMAND} --parallel ${N} --output-on-failure
+    COMMENT "Compile and test with diagnostics positions enabled"
```

Comment:
Sorry - I missed the typo: It should be `Compile and test with diagnostic positions enabled`. (no `s` at the end of "diagnostic")

Reply:
no worries, I just pushed the change.